frontend/www/js/omegaup/components/arena/Runs.vue

```diff
@@ -296,10 +338,43 @@
                   <font-awesome-icon :icon="['fas', 'question-circle']" />
                 </button>
               </td>
+              <td v-if="showProblem" class="text-break-all">
+                <a
+                  href="#runs"
+                  @click.prevent="filterProblem.key = run.alias"
```

Comment:
Esto no va a funcionar correctamente. Tendrías que llenar el objeto completo, eso lo puedes lograr con el método `onEmitFilterChanged` de la siguiente forma:
```suggestion
                  @click="
                    onEmitFilterChanged({
                      filter: 'problem',
                      value: run.alias,
                    })
                  "
```